ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SFLOW_TESTS.yml

```diff
@@ -0,0 +1,118 @@
+---
+# Common AVD group variables
+
+fabric_name: SFLOW_TESTS
+
+mgmt_gateway: 192.168.0.1
+
+fabric_sflow:
+  uplinks: true
+  downlinks: true
+  endpoints: true
+  l3_edge: true
+  core_interfaces: true
+  mlag_interfaces: true
+
+# P2P links between spine switches.
+core_interfaces:
+  p2p_links:
+    - nodes: [ sflow-tests-spine1, sflow-tests-spine2 ]
+      interfaces: [ Ethernet9, Ethernet9 ]
+      include_in_underlay_protocol: false
+      sflow: true
```

Comment:
I think you need one more link without sflow set, so it inherits from the global setting.